backend/applications/api/v1/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+
+from applications.models import Application, SummerBill
+from applications.services import update_or_create_summer_bills_from_api_data
+from companies.api.v1.serializers import CompanySerializer
+
+
+class SummerBillSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SummerBill
+        fields = [
+            "id",
+            "summer_bill_id",
+            "contact_name",
+            "contact_email",
+            "work_postcode",
+            "employee_name",
+            "employee_school",
+            "employee_ssn",
+            "employee_phone_number",
+            "is_unnumbered_summer_bill",
+            "unnumbered_summer_bill_reason",
+        ]
+
+
+class ApplicationSerializer(serializers.ModelSerializer):
+    company = CompanySerializer(required=False, allow_null=True)
+    summer_bills = SummerBillSerializer(many=True, required=False, allow_null=True)
+
+    class Meta:
+        model = Application
+        fields = [
+            "id",
+            "status",
+            "invoicer_name",
+            "invoicer_email",
+            "invoicer_phone_number",
+            "company",
+            "summer_bills",
+        ]
+
+    def update(self, instance, validated_data):
+        validated_data.pop("company", None)
```

Comment:
This could use a comment to better understand why this is being done. At least for a person that does not know the code base well, this is something that would need to be asked or looked up elsewhere to get answered.

Reply:
Yea, I'll try to add a comment about this and refactor the logic to be a bit better.